lib/integrations/openai_base_service.rb

```diff
@@ -75,6 +75,8 @@ def make_api_call(body)
     }
 
     response = HTTParty.post(API_URL, headers: headers, body: body)
-    JSON.parse(response.body)['choices'].first['message']['content']
+    choices = JSON.parse(response.body)['choices']
+
+    choices ? choices.first['message']['content'] : nil
```

Comment:
```suggestion
    choices.present? ? choices.first['message']['content'] : nil
```

will cover more cases like an empty string